File: dojo20100811 - game of life - php/game_of_life.php
<?php

function game_of_life($matriz)
{
    return array(0,0);

}

class Celula{

    public $viva = true;

    public function vizinhosVivos($vizinhos){

        $this->viva = ($vizinhos == 3) || ($vizinhos == 2) && $viva;

    }

}

File: 2016/20160914 - game of life - python/game_of_life.py
def game_of_life(table_of_life):
	

	if len(table_of_life[0]) == 2:
		return [[0, 0]]

	if len(table_of_life[0]) == 3:
		return [[0, 0, 0]]
	
	if len(table_of_life[0]) == 3 and len(table_of_life[0][1] == 1):
		return [[0, 1, 0]]
		

	return [
		[0]
	]

